dashboard/app/helpers/application_helper.rb

```diff
@@ -48,9 +48,15 @@ def check_mark_html
     image_tag(image_url('white-checkmark.png'))
   end
 
-  def activity_css_class(result, submitted=false)
+  def activity_css_class(user_level)
     # For definitions of the result values, see /app/src/constants.js.
-    if submitted
+    result = user_level.try(:best_result)
+
+    if result == Activity::REVIEW_REJECTED_RESULT
```

Comment:
Convention I'm used to (though I don't know what the performance impact is) is to put the order of these in the most common normal case to least common normal case. So submitted would be on top